styles/boringproxy/catppuccin.user.css

```diff
@@ -0,0 +1,292 @@
+	/* ==UserStyle==
+@name           Catpuccin for BoringProxy
+@namespace      github.com/openstyles/stylus
+@version        0.1.0
+@description    Userstyle for BoringProxy Dashboard
+@author         Gandalf-the-Blue
```

Comment:
Can you alter these lines to be similar to the template:
https://github.com/catppuccin/userstyles/blob/main/template/catppuccin.user.css#L2-L9